src/components/StoreMigrator/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
+
+const StoreMigrator: React.FC = () => {
+  const [migrationDone, setMigrationDone] = useState(false)
+  window.addEventListener(
+    'message',
+    (event) => {
+      if (event.data.migrate) {
+        // Update localstorage
+        window.parent.postMessage(
+          {
+            migrateDone: true,
+          },
+          '*',
+        )
+      }
+      if (event.data.migrateDone) {
+        // TODO Turn migration done flag on
+        setMigrationDone(true)
+      }
+    },
+    false,
+  )
+
+  const loadFrame = () => {
+    window.open('http://localhost:3000', 'targetWindow')
```

Comment:
Should we create a special route for this iframe content? That would not load anything except for the necessary code?